shark-graph/src/test/java/shark/ByteArrayTimSortTest.kt

```diff
@@ -0,0 +1,110 @@
+package shark
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import shark.internal.aosp.ByteArrayComparator
+import shark.internal.aosp.ByteArrayTimSort
+import java.lang.IllegalStateException
+import kotlin.random.Random
+
+class ByteArrayTimSortTest {
```

Comment:
Maybe add a link to [TimSort](https://android.googlesource.com/platform/libcore/+/jb-mr2-release/luni/src/main/java/java/util/TimSort.java) for reference?